setup/module/api/policies/cloudtrail_role_policy.json

```diff
@@ -0,0 +1,15 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
```

Comment:
Why two statements here? Your Resource looks the same so couldn't you just update the `Action` property to an array which has both `logs:CreateLogStream` and `logs:PutLogEvents`?

Reply:
You're right, I believe this can be condensed to one Allow statement.

Reply:
:heavy_check_mark: